### ISSUE TITLE
Issue running `check` on 0.24.0-alpha01

### ISSUE THREAD

**Issue by sproctor** [icerockdev/moko-resources#628]
After switching to 0.24.0-alpha01 I get the follow error:

```
* What went wrong:
Some problems were found with the configuration of task ':module:generateMRandroidUnitTest' (type 'GenerateMultiplatformResourcesTask').
  - Gradle detected a problem with the following location: '.../build/generated/moko-resources/androidUnitTest/assets'.
    
    Reason: Task ':module:generateDebugLintModel' uses this output of task ':module:generateMRandroidUnitTest' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed.
```

Adding the following workaround to my gradle build file resolves the issue:
```kotlin
afterEvaluate {
    tasks.named("generateDebugLintModel").configure { dependsOn("generateMRandroidUnitTest") }
    tasks.named("generateDebugLintReportModel").configure { dependsOn("generateMRandroidUnitTest") }
}
```

**Comment by Alex009** [comment]
should be fixed in 0.24.0-alpha-3
in #575

**Comment by sproctor** [comment]
Thanks! It's fixed for me.
